website/style.css

```diff
@@ -1,421 +1,443 @@
 @keyframes fadeInBottom {
-    from {
-        transform: translateY(100px);
-        opacity: 0%;
-    }
-    to {
-        transform: translateY(0px);
-        opacity: 100%;
-    }
+  from {
+    transform: translateY(100px);
+    opacity: 0%;
+  }
+  to {
+    transform: translateY(0px);
+    opacity: 100%;
+  }
 }
+
 @keyframes fadeInRight {
-    from {
-        transform: translateX(100px);
-        opacity: 0%;
-    }
-    to {
-        transform: translateX(0px);
-        opacity: 100%;
-    }
+  from {
+    transform: translateX(100px);
+    opacity: 0%;
+  }
+  to {
+    transform: translateX(0px);
+    opacity: 100%;
+  }
 }
+
 @keyframes fadeInTop {
-    from {
-        transform: translateY(-100px);
-        opacity: 0%;
-    }
-    to {
-        transform: translateY(0px);
-        opacity: 100%;
-    }
+  from {
+    transform: translateY(-100px);
+    opacity: 0%;
+  }
+  to {
+    transform: translateY(0px);
+    opacity: 100%;
+  }
 }
+
 @keyframes fadeInLeft {
-    from {
-        transform: translateX(-100px);
-        opacity: 0%;
-    }
-    to {
-        transform: translateX(0px);
-        opacity: 100%;
-    }
+  from {
+    transform: translateX(-100px);
+    opacity: 0%;
+  }
+  to {
+    transform: translateX(0px);
+    opacity: 100%;
+  }
 }
+
 @keyframes slideInBottom {
-    from {
-        transform: translateY(100px);
-    }
-    to {
-        transform: translateY(0px);
-    }
+  from {
+    transform: translateY(100px);
+  }
+  to {
+    transform: translateY(0px);
+  }
 }
+
 @keyframes slideInRight {
-    from {
-        transform: translateX(100px);
-    }
-    to {
-        transform: translateX(0px);
-    }
+  from {
+    transform: translateX(100px);
+  }
+  to {
+    transform: translateX(0px);
+  }
 }
+
 @keyframes slideInTop {
-    from {
-        transform: translateY(-100px);
-    }
-    to {
-        transform: translateY(0px);
-    }
+  from {
+    transform: translateY(-100px);
+  }
+  to {
+    transform: translateY(0px);
+  }
 }
+
 @keyframes slideInLeft {
-    from {
-        transform: translateX(-100px);
-    }
-    to {
-        transform: translateX(0px);
-    }
-}
-@keyframes flexAnimation{
-    from{flex:0;}
-    to{flex:1;}
+  from {
+    transform: translateX(-100px);
+  }
+  to {
+    transform: translateX(0px);
+  }
+}
+
+@keyframes flexAnimation {
+  from {
+    flex: 0;
+  }
+  to {
+    flex: 1;
+  }
 }
 
 @keyframes fadeIn {
-    
 }
 
 body {
-    background-image: linear-gradient(-90deg, #181818, black);
+  background-image: linear-gradient(-90deg, #181818, black);
 }
 
-p, span, ul, li, h1, h2 {
-    z-index: 3;
+p,
+span,
+ul,
+li,
+h1,
+h2,
+img {
+  z-index: 4;
 }
 
 * {
-    box-sizing: border-box;
-    margin: 0;
-    color: white;
-    font-family: 'Roboto Mono', monospace;;
-    font-weight: normal;
-    line-height: 30px;
-    text-decoration: none;
+  box-sizing: border-box;
+  margin: 0;
+  color: white;
+  font-family: "Roboto Mono", monospace;
+  font-weight: normal;
```

Comment:
nn this is default [font-weight](https://www.w3schools.com/cssref/pr_font_weight.asp)

Reply:
fc07956